ppr-ui/src/interfaces/steps-interface.ts

```diff
@@ -7,7 +7,7 @@ export interface StepIF {
   icon: string
   text: string
   to: RouteNames
-  component: Component
+  component: Component | string
```

Comment:
What are examples of a sting component?

Reply:
PPR Step configurations pass string values, which is technically "acceptable" because it's accepting the component `name` value.